manifests/w/WatchGuardTechnologies/WatchGuardSystemManager/12.7.1/WatchGuardTechnologies.WatchGuardSystemManager.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.1.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: WatchGuardTechnologies.WatchGuardSystemManager
+PackageVersion: 12.7.1
+PackageLocale: en-US
+Publisher: WatchGuard Technologies, Inc.
+PackageName: WatchGuard System Manager 12.7.1
+License: (c) 2008-2018 WatchGuard Technologies, Inc.
+ShortDescription: WatchGuard System Manager 12.7.1 Setup
```

Comment:
```
ShortDescription: WatchGuard System Manager 12.7.1 Setup
```
Please write a better description of what the application is about.

Reply:
Modified description to be descriptive.